ocaml/xapi/xapi_xenops.ml

```diff
@@ -14,6 +14,8 @@
 
 module D=Debug.Make(struct let name="xenops" end)
 open D
+module StringSet = Set.Make(struct type t = string let compare = compare end)
+module VMSet = Set.Make(struct type t = [ `VM ] API.Ref.t let compare = compare end)
```

Comment:
It looks like you are not using `VMSet`.

Reply:
True! I had previously used it, but realised that the case where I was using it was just converting a list to a set and back again, so removed where it was used, forgot to remove the module declaration